test/tests/meshgenerators/break_mesh_by_block_generator/tests

```diff
@@ -70,4 +70,19 @@
     mesh_mode = 'REPLICATED'
     recover = false
   [../]
+
+  [./hanging_nodes]
+    type = 'Exodiff'
+    input = 'hanging_nodes_parallel.i'
+    cli_args = '--mesh-only'
+    exodiff = 'hanging_nodes_parallel_in.e'
+    requirement = 'MOOSE shall be able to create separate blocks in a mesh in parallel'
+    design = 'meshgenerators/BreakMeshByBlockGenerator.md'
+    issues = '#15793'
+    mesh_mode = 'REPLICATED'
+    min_parallel = 4
+    max_parallel = 5
+    recover = false
+    method = 'dbg'
```

Comment:
```suggestion
```
Go ahead and remove this. It is a good idea to run it in opt too even though your assertion doesn't get hit there.